rvv-intrinsic-generator/Makefile

```diff
@@ -106,9 +106,9 @@ define gen_docs
 endef
 
 define gen_tests
-	rm -f $(1)/*.c
+	rm $(1)/*.c $(1)/*.C -f
```

Comment:
I suppose we don't generate source files like "*.C" anymore, @eopXD  can help to confirm about this.